src/azure-cli/azure/cli/command_modules/rdbms/tests/latest/test_rdbms_flexible_commands.py

```diff
@@ -467,3 +468,448 @@ def _test_flexible_server_replica_mgmt(self, database_engine, resource_group):
                  .format(database_engine, resource_group, replicas[0]), checks=NoneCheck())
         self.cmd('{} flexible-server delete -g {} --name {} --yes'
                  .format(database_engine, resource_group, replicas[1]), checks=NoneCheck())
+
+
+class FlexibleServerVnetMgmtScenarioTest(ScenarioTest):
+
+    postgres_location = 'eastus'
+    mysql_location = 'westus2'
+    '''
+    @AllowLargeResponse()
+    @ResourceGroupPreparer(location=mysql_location)
+    def test_investigation(self, resource_group):
+        self.helper('mysql', resource_group)
+
+    '''
+    @AllowLargeResponse()
+    @ResourceGroupPreparer(location=postgres_location)
+    @live_only()
```

Comment:
could you remove @live_only() and let me see the error in CI system when playback?

Reply:
> could you remove @live_only() and let me see the error in CI system when playback?

@Juliehzl : Removed. The test test_investigation2() is the concerning test. I have pinged you offline about the details.

Reply:
Was able to fix the open issues. Hence, removed the tests I had added for investigation.